docs/getting-started/integrating-codacy-with-visual-studio-code.md

```diff
@@ -1,90 +1,87 @@
+
+!!! info "To use this extension you need a [Codacy account](https://www.codacy.com/signup-codacy)"
 
 ## Installing the Codacy VS Code extension
 
-!!! note
-    Before installing the extension, make sure you meet the following requirements:
+1.  Make sure that the repository you’re working on is analyzed by Codacy and that you have at least [Repository Read permissions](../organizations/roles-and-permissions-for-organizations.md).
+
+    !!! tip
+        If this is your first time using Codacy, see [how to add and analyze your first repository](./codacy-quickstart.md#adding-your-first-repository).
 
-    -   You have a [Codacy account](https://www.codacy.com/signup-codacy).
-    -   The repository you’re working on has been [added to Codacy Cloud](../organizations/managing-repositories.md#adding-a-repository) and analyzed.
-    -   You have at least [Repository Read permissions](../organizations/roles-and-permissions-for-organizations.md) for the repository you’re working on.
+1.  Install the extension from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=codacy-app.codacy) or through the [Extensions view in VS Code](https://code.visualstudio.com/docs/editor/extension-marketplace#_browse-for-extensions).
 
-You can install the extension from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=codacy-app.codacy) or through the [Extensions view](https://code.visualstudio.com/docs/editor/extension-marketplace#_browse-for-extensions) in Visual Studio Code. Alternatively, you can [download the latest release as a VSIX package](https://github.com/codacy/codacy-vscode-extension/releases) and install it manually.
+    Alternatively, you can install it manually by [downloading the latest release as a VSIX package](https://github.com/codacy/codacy-vscode-extension/releases).
 
-## Fetching pull request quality and coverage data from Codacy
+## Getting pull request quality and coverage data
+
+To see Codacy quality and coverage data for the pull request you're working on, follow these steps:
 
 1.  Open the repository directory in VS Code.
 
     !!! note
-        Make sure that you’ve [added the repository to Codacy](../organizations/managing-repositories.md#adding-a-repository).
+        If the repository isn't on Codacy yet, [add it to Codacy](../organizations/managing-repositories.md#adding-a-repository) first.
 
-1.  Open the main view by clicking the **Codacy logo** in the status bar or the **Codacy tab** in the activity bar.
+1.  Open the main view by clicking the **Codacy logo** in the activity bar or the **Codacy tab** in the status bar.
 
     ![Codacy main view](images/codacy-vscode-extension-sign-in.png)
 
-1.  If you’re not signed in, click the **Sign In** button to authorize VS Code on Codacy.
+1.  If you’re not signed in, click the **Sign in** button to authorize VS Code on Codacy.
 
-1.  Check out the relevant branch.
+1.  Check out the branch you're working on.
```

Comment:
Maybe we can avoid the admonition by including the information within the step.

Also, I believe it makes sense to move this step to step 2 of the procedure, right after opening the repository directory in VS Code and before opening the extension. 🤔 

Thoughts?

```suggestion
1.  Check out the branch associated with the open pull request you're working on.
```

Reply:
Sounds good! [Done](https://github.com/codacy/docs/pull/1863/commits/ef6dea95b1daaa2883400cada8f0cda27ca99638) 👍